.github/tests/nested/values.yaml

```diff
@@ -0,0 +1,16 @@
+spire-server:
+  enabled: true
+  upstreamAuthority:
+    spire:
+      enabled: true
+      server:
+        address: spire-server.nested-deps
+  controllerManager:
+    enabled: true
+    identities:
+      spiffeIDTemplate: spiffe://{{ .TrustDomain }}/k8s/example-cluster/ns/{{ .PodMeta.Namespace }}/sa/{{ .PodSpec.ServiceAccountName }}
```

Comment:
Do we have to override this? Is the default value not good enough?

Reply:
We dont have to override it, no. Was testing it, as when you want to use nested spire, your likely to want to have multiple k8s clusters and each would hae to have a unique cluster name like example-cluster in its template. It could be removed. But its kind of useful documentation too, so not sure where it should go.

Reply:
We may also want to template that out by default so whatever the clustername is set to, will automatically get it set in the template so this would pretty much never need an override?